signature-specification.md

```diff
@@ -81,7 +81,10 @@ Examples:
 {
    "mediaType": "application/vnd.oci.image.manifest.v1+json",
    "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
-   "size": 16724
+   "size": 16724,
+   "annotations": {
+      "wabbit-networks.io.buildId": "123",  // user defined signed attribute.
```

Comment:
Let's mark this code block using `jsonc` instead of `json` for better syntax highlighting for comments.